<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCase</class>
 <widget class="QDialog" name="CreateCase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>1150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>320</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create a case</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="WName" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LEName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>199</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Directory">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushDir">
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEDirName">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="4">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Mesh">
       <property name="text">
        <string>Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushFichier">
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEFileName">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="5">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="GBTypeConf">
     <property name="title">
      <string>Conformity type</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="RBConforme">
        <property name="text">
         <string>Conformal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBNonConforme">
        <property name="text">
         <string>Non conformal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QGroupBox" name="GBTypeBoun">
     <property name="title">
      <string>Boundary type</string>
     </property>
     <layout class="QHBoxLayout" name="_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="RBBoundaryNo">
        <property name="text">
         <string>No boundary</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBBoundaryCAO">
        <property name="text">
         <string>CAO</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBBoundaryNonCAO">
        <property name="text">
         <string>Non CAO</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="CBBoundaryD">
       <property name="text">
        <string>Discrete boundary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CBBoundaryA">
       <property name="text">
        <string>Analytical boundary</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="GBBoundaryC">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>CAO</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="PBBoundaryCAOEdit">
        <property name="text">
         <string>Edit</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="PBBoundaryCAOHelp">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="CBBoundaryCAO">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PBBoundaryCAONew">
        <property name="text">
         <string>New</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QGroupBox" name="GBBoundaryD">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Discrete boundary</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="CBBoundaryDi">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="PBBoundaryDiEdit">
        <property name="text">
         <string>Edit</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="PBBoundaryDiHelp">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PBBoundaryDiNew">
        <property name="text">
         <string>New</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0" colspan="7">
    <widget class="QGroupBox" name="GBBoundaryA">
     <property name="minimumSize">
      <size>
       <width>548</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Analytical boundary</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="TWBoundary">
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <column>
         <property name="text">
          <string>a_virer</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="PBBoundaryAnEdit">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="PBBoundaryAnNew">
          <property name="text">
           <string>New</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="PBBoundaryAnHelp">
          <property name="text">
           <string>Help</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>239</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="CBAdvanced">
     <property name="text">
      <string>Advanced options</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QGroupBox" name="GBAdvancedOptions">
     <property name="title">
      <string>Advanced options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="CBPyramid">
        <property name="text">
         <string>Authorized pyramids</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="GBConforme">
        <property name="title">
         <string>Conformity +</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="RBStandard">
             <property name="text">
              <string>Standard</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RBBox">
             <property name="text">
              <string>Box</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RBNC1NpA">
             <property name="text">
              <string>1 node per edge</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RBNCQuelconque">
             <property name="text">
              <string>Free</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="GBFormat">
           <property name="title">
            <string>Format</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBMED">
              <property name="text">
               <string>MED</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBSaturne">
              <property name="text">
               <string>Saturne</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBSaturne2D">
              <property name="text">
               <string>Saturne 2D</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="4" colspan="3">
    <spacer name="spacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>128</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <spacer name="spacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>239</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="Comment">
     <property name="text">
      <string> No comment.</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QGroupBox" name="GroupButtons">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="4">
       <widget class="QPushButton" name="buttonHelp">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonApply">
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonOk">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="15" column="3" colspan="2">
    <spacer name="spacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>128</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>WName</zorder>
  <zorder>GBTypeConf</zorder>
  <zorder>GBBoundaryD</zorder>
  <zorder>GBBoundaryA</zorder>
  <zorder>CBAdvanced</zorder>
  <zorder>GBAdvancedOptions</zorder>
  <zorder>Comment</zorder>
  <zorder>GroupButtons</zorder>
  <zorder>GBBoundaryC</zorder>
  <zorder>GBTypeBoun</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
